scripts/examples/python/XenAPI/XenAPI.py

```diff
@@ -65,6 +65,14 @@
     import http.client as httplib
     import xmlrpc.client as xmlrpclib
 
+try:
+    from opentelemetry import propagate
+    from opentelemetry.trace.propagation import set_span_in_context, get_current_span
+    otel = True
+
+except ImportErr:
```

Comment:
pylint says this should be `ImportError`. I recommend testing the code with instrumentation disabled to check it still works.